.editorconfig

```diff
@@ -100,6 +100,9 @@ dotnet_style_prefer_is_null_check_over_reference_equality_method = true:error
 #prefer braces
 csharp_prefer_braces = true:error
 
+#do not prefer readonly fields
+dotnet_style_readonly_field = false:error
```

Comment:
It is not working as I expected. I though it will gives error if I define a readonly field but not.

This removes the suggestion to make fields readonly if possible (set only from constructor). But I can do readonly fields without any error. If you agree then the comment should be:
`#do not suggest readonly field`

Reply:
Yes it removes the suggestion to add `readonly` modifier.
In #1901 @nopara73 said https://github.com/zkSNACKs/WalletWasabi/pull/1901#issuecomment-511202343, so maybe we should remove `readonly` everywhere.